centos_ci.sh

```diff
@@ -103,14 +104,7 @@ function setup_kvm_docker_machine_driver() {
   echo 'CICO: Setup KVM docker-machine driver setup successfully'
 }
 
-function install_and_setup_golang() {
-  # Install Go 1.7
-  curl -LO https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz
```

Comment:
Would we not just download the appropriate 1.8 version here?

Reply:
@hferentschik No now we are consuming version directly from official repository.

Reply:
But here we pin the version and it is clear what is used. If we just use the latest from the repo, we don't really know what's used.